### PR TITLE
[DX-1461] EDP 1.10.0 release notes

### PR DESCRIPTION
**Pull request by tbuchaillot**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>
https://tyktech.atlassian.net/browse/DX-1461
---

### Preview Link

<be>

**Release notes**:
https://deploy-preview-4909--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0/

**Configuration options**: 
https://deploy-preview-4909--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/deploy/configuration/#portal_database_max_open_connections

https://deploy-preview-4909--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/deploy/configuration/#portal_database_max_idle_connections

https://deploy-preview-4909--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/deploy/configuration/#portal_database_connection_max_lifetime

https://deploy-preview-4909--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/deploy/configuration/#portal_assets_cache_disable
### Description

<be>

EDP 1.10.0 Release notes.

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

### **PR Type**
Documentation, Enhancement

___

### **Description**
- Added new configuration option `PORTAL_ASSETS_CACHE_DISABLE` to disable the assets cache for themes stored in the database.
- Added new database connection settings: `MaxOpenConnections`, `MaxIdleConnections`, and `ConnectionMaxLifetime` to manage database connections lifecycle.
- Added release notes for Tyk Enterprise Developer Portal v1.10.0, documenting new features, bug fixes, and enhancements.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>configuration.md</strong><dd><code>Add new configuration options for assets cache and database </code><br><code>connections</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md
<li>Added new configuration option <code>PORTAL_ASSETS_CACHE_DISABLE</code>.<br> <li> Added new database connection settings: <code>MaxOpenConnections</code>, <br><code>MaxIdleConnections</code>, and <code>ConnectionMaxLifetime</code>.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4909/files#diff-bff942701e76bae5b357af2a798c59862dff033909d36d1d2bba2d878add91dd">+28/-1</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>portal-1.10.0.md</strong><dd><code>Add release notes for Tyk Enterprise Developer Portal v1.10.0</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md
<li>Added release notes for Tyk Enterprise Developer Portal v1.10.0.<br> <li> Documented new features, bug fixes, and enhancements.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4909/files#diff-8fd95d3e938998e3a78a586191b42b9cb2269e937fc683a9801a2983912af398">+77/-0</a>&nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Theme cache. 
+- Configuration options for database connections.
+
+#### Upgrade instructions
+If you are on 1.9.0 or an older version we advise you to upgrade ASAP directly to this release.
+
+To upgrade the portal's theme please follow the [upgrade instructions]({{< ref "product-stack/tyk-enterprise-developer-portal/upgrading/theme-upgrades" >}}) for the portal's themes.
+
+## Download
+- [Docker image v1.9.0](https://hub.docker.com/r/tykio/portal/tags?page=&page_size=&ordering=&name=v1.10.0)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]:
```suggestion
- [Docker image v1.10.0](https://hub.docker.com/r/tykio/portal/tags?page=&page_size=&ordering=&name=v1.10.0)
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+tags: ["Developer Portal", "Release notes", "changelog", "v1.10.0"]
+menu:
+main:
+parent: "Release Notes"
+weight: 7
+---
+
+**Licensed Protected Product**
+
+##### Release Date 27 Jun 2024
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md:13]: Should this be 1 July 2024?

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+This release has no breaking changes.
+
+#### Future breaking changes
+This release doesn’t introduce future breaking changes.
+
+#### Deprecations
+There are no deprecations in this release.
+
+## Release Highlights
+The 1.10.0 addresses twenty high-priority bugs and vulnerabilities bugs and introduces three new capabilities:
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]:
```suggestion
The 1.10.0 addresses twenty high-priority bugs and vulnerabilities and introduces three new features:
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+## Changelog
+#### Added
+- Added OAS APIs support. 
+- Added an assets cache for improved performance on database-backed themes. This speeds up the portal's pages loading time by 30%. It's enabled by default and you can disable using `PORTAL_ASSETS_CACHE_DISABLE`.
+- Added three new configuration options to manage database connections lifecycle: `PORTAL_DATABASE_MAX_OPEN_CONNECTIONS`, `PORTAL_DATABASE_MAX_IDLE_CONNECTIONS` and `PORTAL_DATABASE_CONNECTION_MAX_LIFETIME`.
+
+#### Fixed
+- Fixed the bug where `PORTAL_SESSION_LIFETIME` was calculated in minutes instead of seconds.
+- Fixed the bug where accesss requests were not removed when an application is deleted.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]:
```suggestion
- Fixed the bug where access requests were not removed when an application is deleted.
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- Fixed the bug where stoplight library was blocking the portal's startup if it's not avaialble.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]:
```suggestion
- Fixed the bug where stoplight library was blocking the portal's startup if it's not available.
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- Fixed the bug where browsing into API Product throws an error when Baseline URL is provided in provider section.
+- Fixed the bug where it was possible to create a new access requests from the admin dashboard.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: How was this an issue, what problem did it cause?

```suggestion
- Fixed the bug where it was possible to create new access requests from the admin dashboard.
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- Fixed the bug where the portal `/ready` probe was not taking into consideration the bootstrap and tables automigration process.
+- Fixed the bug where sometimes, the plan added to the cart was not updated after a product change.
+- Fixed the bug where it was not possible to delete an application that was provisioned with an access request created through the API.
+- Fixed the bug where users where not able to submit the cart from parallel submission (two different tabs or browsers). 
+- Fixed the bug where creating an app was not possible when there was no DCR scope specified for the Product but there was a scope specified for the Plan.
+- Fixed the bug where the portal logout was not clearing browser user data and logging the user out completely.
+- Fixed the bug where it was not possible to delete non authToken apps from the developer portal when approved products and plans are removed.
+- Fixed the bug where it was not possible to download the theme without adding an extra `/` to the URL.
+- Fixed the bug where carts submissions where affecting other users carts.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: Add explanation how carts submissions were affecting other user carts

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+This release doesn’t introduce future breaking changes.
+
+#### Deprecations
+There are no deprecations in this release.
+
+## Release Highlights
+The 1.10.0 addresses twenty high-priority bugs and vulnerabilities bugs and introduces three new capabilities:
+- OAS APIs support.
+- Theme cache. 
+- Configuration options for database connections.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md:28]: Briefly explain PORTAL_DATABASE_MAX_OPEN_CONNECTIONS, PORTAL_DATABASE_MAX_IDLE_CONNECTIONS and PORTAL_DATABASE_CONNECTION_MAX_LIFETIME and link to accompanying documentation. If new environment variables then documentation should be added [here](https://tyk.io/docs/product-stack/tyk-enterprise-developer-portal/deploy/configuration/)

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+- [Docker image v1.10.0](https://hub.docker.com/r/tykio/portal/tags?page=&page_size=&ordering=&name=v1.10.0)
+  - ```bash
+    docker pull tykio/portal:v1.10.0
+    ```
+- [The default theme package](https://github.com/TykTechnologies/portal-default-theme/releases/tag/1.10.0)
+
+## Changelog
+#### Added
+- Added OAS APIs support. 
+- Added an assets cache for improved performance on database-backed themes. This speeds up the portal's pages loading time by 30%. It's enabled by default and you can disable using `PORTAL_ASSETS_CACHE_DISABLE`.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: To disable PORTAL_ASSETS_CACHE_DISABLE=True | PORTAL_ASSETS_CACHE_DISABLE=true ?

If new env var then docs needs adding [here](https://tyk.io/docs/product-stack/tyk-enterprise-developer-portal/deploy/configuration/)

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM @tbuchaillot, made minor suggestions and left comments relating to how some of the bug caused an issue.

If the environment variables are new relating to the connection and cache then will need adding and linked to [here](https://tyk.io/docs/product-stack/tyk-enterprise-developer-portal/deploy/configuration/)

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md (hunk 6 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: @dcs3spp it was an operation that shouldn't be possible at all 😆 that's why it was classified as a bug. Commited the suggestion, but won't go into more details.

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md (hunk 2 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md:13]: Hopefully we can release it today 😆 I'll modify the date accordingly

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md (hunk 7 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: added!

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md (hunk 9 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md]: @dcs3spp the new env vars was already there, added a link to that in the release notes 👌

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md (hunk 8 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.10.0.md:28]: @dcs3spp the new env vars were already there, linked them to the release notes 👌

**Review by dcs3spp — APPROVED** [review]
@tbuchaillot PR approved, LGTM! @MFCaballero could you peer review and if all ok approve.

Once peer approved will merge and release

**Review by edsonmichaque — APPROVED** [review]
LGTM

**Review by MFCaballero — APPROVED** [review]
lgtm

**Pull request merged**